### PR TITLE
[DPE-1086] Add new extensions/plugins

### PR DESCRIPTION
**Pull request by TakoB222**
Added support for new extensions/plugins:

- icu_ext
- address_standardizer
- address_standardizer
- address_standardizer_data_us
- bool_plperl
- hll
- hstore
- hypopg
- ip4r
- jsonb_plperl
- orafce
- pg_similarity
- pg_trgm
- plperl
- pltcl
- postgis
- postgis_raster
- postgis_tiger_geocoder
- postgis_topology
- prefix
- rdkit
- tds_fdw
- unaccent

### CODE REVIEW

File: lib/charms/postgresql_k8s/v0/postgresql.py
Comment (dragomirp) [inline lib/charms/postgresql_k8s/v0/postgresql.py:1]: All the other changes to the lib must be the exact same here. Best to copy it from the other PR.

Comment (TakoB222) [reply lib/charms/postgresql_k8s/v0/postgresql.py:1]: Yeah, fixed

**Review by taurus-forever — APPROVED** [review]

**Comment by taurus-forever** [comment]
Please add a ticket number and PR description. Tnx!

**Review by marceloneppel — APPROVED** [review]

**Comment by dragomirp** [comment]
Tested locally and suggested some changes to the rock. We shouldn't need anything else in this PR once we update the Rock.

**Review by dragomirp — APPROVED** [review]

**Review by marceloneppel — APPROVED** [review]

**Pull request merged**